resources/definitions/store/lobby.gd

```diff
@@ -0,0 +1,80 @@
+class_name Lobby
+extends DictResource
+
+signal player_removed(player_id)
+
+export(String) var id = null
+export(Resource) var owner = null # Player
+export(Dictionary) var players
+export(Resource) var option = null #= LobbyOption.new()
+
+
+func _init(dict = null, player : Player = null).(dict):
+	if player:
+		relink_player(player)
+
+
+func load_dict(dict):
+	if dict == null:
+		return
+	.load_dict(dict)
+	if (not dict is Dictionary or dict.has("players")) and dict.players != null:
+		players.clear()
+		for player in dict.players:
+			if player is String:
+				players[player.id] = Player.new({"id" : player})
+			else:
+				players[player.id] = Player.new(player)
+	if not dict is Dictionary or dict.has("owner"):
+		if dict.owner is String:
+			owner = Player.new({"id" : dict.owner})
+		else:
+			owner = Player.new(dict.owner)
+		players[owner.id] = owner # relink owner to the array of player
```

Comment:
Not sure of this due to your comment but worth the try :D !
```suggestion
		players[owner.id] = Player.new({"id" : dict.owner} if dict.owner is String else dict.owner) # relink owner to the array of player
```

Reply:
`owner` has to be set, I used a ternary to set `owner` first